docs/device/install.md

```diff
@@ -6,11 +6,17 @@
 
 1. If you have got __`navtunnel`__ installed -> [uninstall navtunnel](#uninstall-navtunnel).
 2. [Install the Kolide agent](#install-kolide-agent).
-3. Write in the terminal: `curl https://device.nais.io/install.sh | bash` \(you might be prompted for your laptop-account's password to unlock `sudo`\).
-4. Turn on your freshly installed `naisdevice` app.
+3. Install [Homebrew](https://brew.sh/) unless yoy already have it.
+4. Write in the terminal: 
+```bash
+brew tap nais/tap
+brew install naisdevice
+``` 
+(you'll might be prompted for your laptop-account's password to unlock `sudo`).
```

Comment:
The resulting action is the installation of a .pkg no? In that case the pkg should/will prompt for credentials and so "you'll might" should be replaced with "you will".